src/models/tiles/text/text-plugin-info.ts

```diff
@@ -26,12 +36,11 @@ export function getTextPluginInfo(id: string) {
 
 // TODO: perhaps this should only add the plugins that have been configured
 // as tools by the app-config.
-export function getTextPluginInstances(textContent: TextContentModelType) {
-  // FIXME: need to figure out what this should be
-  const pluginInstances:  any[] = []; // FIXME type
+export function createTextPluginInstances(textContent: TextContentModelType) {
+  const pluginInstances:  Record<string, ITextPlugin> = {};
```

Comment:
I should convert this to 
```suggestion
  const pluginInstances:  Record<string, ITextPlugin|undefined> = {};
```
To defensively catch the case when the code tries to find a plugin that doesn't exist.